nix/package.nix

```diff
@@ -0,0 +1,48 @@
+{ pkgs
+, stdenv
+, lib
+, buildPythonPackage
+
+, six
+, filetracker
+, simplejson
+, twisted
+, urllib3
+, sortedcontainers
+, bsddb3
+
+, pytest
+, pytest-runner
+, pytest-timeout
+
+, ...
+}:
+
+buildPythonPackage {
+  pname = "sioworkers";
+  version = "1.0";
+
+  src = builtins.path {
+    path = ./..;
+    filter = path: type: builtins.match ".*/nix" path == null;
+  };
+
+  doCheck = false;
+
+  nativeBuildInputs = [
```

Comment:
If someday upstream switches to 64-bit like us, we will need to add gcc,fpc and glibc.static here